README.md

```diff
@@ -84,6 +85,19 @@ Binary will be created as `<path to your test file.nim>.bin` under the `build` d
 make test/tests/common/test_enr_builder.nim
 ```
 
+## Formatting
+
+Nim files are expected to be formatted using the [`nph`](https://github.com/arnetheduck/nph) version present in `vendor/nph`.
+
+You can easily format file with the `make nph/<relative path to nim> file` command.
```

Comment:
Shall we have an example of how to format a particular file? That would help to understand the command :)

( maybe it sounds more natural having `make nph <relative-path-to-nim-file>` )